spec/example_event_spec.rb

```diff
@@ -0,0 +1,135 @@
+require "spec_helper"
+
+# This is an illustration of the general testing approach,
+# as found in the [.NET model tests](https://github.com/openactive/OpenActive.NET/blob/master/OpenActive.NET.Test/EventTest.cs)
+# For better written tasks, please see the other unit tests.
+#
+# @see https://github.com/openactive/models-lib/tree/master/test-description For the test description
+RSpec.describe "Example Event" do
+  # it "returns expected json ld" do
+  # Test that serialized serialized event returns expected JSON-LD.
+  #
+  # @dataProvider sessionSeriesProvider
+  # @return void
+  # def compare_event_jsonld($event, $json)
+  # {
+  #     $classname = "\\".get_class($event);
+  #
+  #     $jsonEvent = $classname::serialize($event);
+  #
+  #     $this->assertEquals(
+  #         json_decode($json, true),
+  #         json_decode($jsonEvent, true)
+  #     );
+  # }
+
+  # Test event accessors.
+  #
+  # @dataProvider sessionSeriesProvider
+  # @return void
+  # public function testSerializeEventAccessor($event)
+  # {
+  #     $this->assertSame(
+  #         "Santa Clara City Library, Central Park Library",
+  #         $event->getLocation()->getName()
+  #     );
+  # }
+
+  # This test does not make sense.
+  #
+  # @return void
+  # public function testSerializeOfferCast()
```

Comment:
Not sure if you want to remove this - I think I ended up removing it in the PHP implementation.

Reply:
Even if it is kept it should not include commented out PHP code

Reply:
Removed.